OpenEdXMobile/res/values/text_styles.xml

```diff
@@ -210,4 +210,27 @@
         <item name="android:textColor">@color/infoBase</item>
     </style>
 
+    <style name="profile_field_title" parent="Widget.AppCompat.TextView">
+        <item name="android:layout_width">0dp</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_weight">1</item>
+        <item name="android:fontFamily">?attr/fontSemiBold</item>
+        <item name="android:lineHeight">@dimen/widget_margin_double</item>

```

Comment:
Update the line into : 
`        <item name="android:lineHeight" tools:ignore="NewApi">@dimen/widget_margin_double</item>
`